src/vm/prestub.cpp

```diff
@@ -2108,6 +2139,17 @@ EXTERN_C PCODE STDCALL ExternalMethodFixupWorker(TransitionBlock * pTransitionBl
                 pCode = PatchNonVirtualExternalMethod(pMD, pCode, pImportSection, pIndirection);
             }
         }
+
+#if defined (FEATURE_JIT_PITCHING)
```

Comment:
Similar to above, lets reduce this down to a single call out:

    #if defined (FEATURE_JIT_PITCHING)
        MarkMethodNotPitchingCandidate(pMD);
    #endif

Your call on naming and whether or not this method should be the same or different than the one above (I note one added to the NotForPitching set, the other didn't).